src/instr/instr_interface.cpp

```diff
@@ -176,7 +176,7 @@ void TRACE_declare_mark(const char *mark_type)
  * \see TRACE_mark
  */
 void TRACE_declare_mark_value_with_color (const char *mark_type, const char *mark_value, const char *mark_color)
-{
+{ paje_value pj_value;
```

Comment:
Could you please follow the advices of http://simgrid.gforge.inria.fr/simgrid/latest/doc/uhood_tech.html (in particular the section about the coding standards and clang-format)?